From d4026c4839f3f5779fdba9213631817977125f9a Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Fri, 16 Apr 2021 12:58:05 +0200
Subject: [PATCH] foris: refactor HOST_PYTHON3_PACKAGE_BUILD_DEPENDS

Based on the upstream change versioning is used in host-pip-requirements
inside packages feed, so we can drop it here
---
 web/foris/foris/Makefile | 13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

diff --git a/web/foris/foris/Makefile b/web/foris/foris/Makefile
index d0affd8c2..0b0877699 100644
--- a/web/foris/foris/Makefile
+++ b/web/foris/foris/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=foris
 PKG_VERSION:=101.1.1
-PKG_RELEASE:=4
+PKG_RELEASE:=5
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/foris/foris.git
@@ -21,11 +21,12 @@ PKG_LICENSE:=LICENSE
 PKG_LICENSE_FILES:=GPL-3.0-or-later
 
 HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:= \
-	pytz==2018.9 \
-	Babel==2.6.0 \
-	libsass==0.14.5 \
-	MarkupSafe==1.1.1 \
-	Jinja2==2.10
+	pytz \
+	Babel \
+	libsass \
+	MarkupSafe \
+	Jinja2 \
+
 PKG_BUILD_DEPENDS:= \
 	compass/host \
 	ruby-breakpoint/host
-- 
2.25.1

